sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-generation.service.ts

```diff
@@ -322,4 +323,24 @@ export class ECertGenerationService {
       },
     });
   }
+
+  /**
+   * Get the lifetime maximums of CSLP of the student which is used in the e-Cert generation.
+   * @param assessmentId disbursement assessment id.
+   * @param entityManager used to execute the commands in the same transaction.
+   * @returns lifetime maximums of CSLP.
+   */
+  async getCSLPLifetimeMaximums(
```

Comment:
Not a blocker but I would expect this kind of method to be inside a `StudentAssessmentService` inside integrations.